frontend/src/components/Displays/RobotDisplays/RobotStatusIcon.tsx

```diff
@@ -33,11 +33,14 @@ export const RobotStatusChip = ({ status, flotillaStatus, isarConnected, itemSiz
     let iconColor = tokens.colors.text.static_icons__default.hex
     let statusIcon = Icons.CloudOff
     switch (status) {
+        case RobotStatus.Home:
+        case RobotStatus.ReturningHome:
         case RobotStatus.Available: {
             statusIcon = Icons.Successful
             iconColor = tokens.colors.interactive.success__resting.hex
             break
         }
+        case RobotStatus.UnkownStatus:
```

Comment:
Unknown status should be error as opposed to busy shouldn't it?

Reply:
Yeah, agreed